core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SequencingParameters.java

```diff
@@ -0,0 +1,39 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+import java.util.Date;
+
+import com.eaglegenomics.simlims.core.User;
+
+public interface SequencingParameters {
+  public Long getId();
+
+  public void setId(Long id);
+
+  public String getName();
+
+  public void setName(String string);
+
+  public Long getPlatformId();
```

Comment:
I believe this field is only for Hibernate purposes since the Platform is loaded via old SQL DAO? If so, Javadoc mention of this would be good. setPlatformId could also be removed to further prevent use of the field since Hibernate uses reflection instead of the setter anyway